src/utils/domUtils.js

```diff
@@ -138,6 +138,24 @@ function getPosition(elem, offsetParent) {
 }
 
 /**
+ * Get an element's size
+ *
+ * @param {HTMLElement} elem
+ * @returns {{width: number, height: number}}
+ */
+function getSize(elem) {
+  if (typeof elem.getBoundingClientRect !== 'undefined') {
```

Comment:
This is not the right check. Non-compliant browsers may return an object that lacks the `width` and `height` entries.